gadgets/top_process/README.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: top_process
+sidebar_position: 0
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# top_process
+
+The top_process gadget periodically reports process statistics, including CPU usage, memory usage, and other process information.
+
+## Getting started
+
+Running the gadget:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/top_process:%IG_TAG%
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ sudo ig run ghcr.io/inspektor-gadget/gadget/top_process:%IG_TAG%
+        ```
+    </TabItem>
+</Tabs>
+
+## Guide
+
+The top_process gadget provides detailed information about running processes, including:
+
+- Process ID (PID)
+- Parent Process ID (PPID)
+- Command name
+- CPU usage percentage
+- Memory usage (RSS and virtual)
+- Thread count
+- Process state
+- User ID
+- Process start time
+- Mount namespace ID
+
+It also adds environment enrichments (container/k8s information), if possible.
+
+You can sort the output by any field, for example by CPU usage:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/top_process:%IG_TAG% --sort -cpuUsage --max-entries 12
```

Comment:
any idea why the mountnsid is shown? Shouldn't it be hidden by default? Smae for starttime and starttimestr, can we hide one of them?